mcc/odin/odin/mock_rds_client_test.go

```diff
@@ -175,6 +175,43 @@ func (m *mockRDSClient) AddSnapshots(
 	)
 }
 
+// CreateDBSnapshot mocks rds.CreateDBSnapshot.
+func (m *mockRDSClient) CreateDBSnapshot(
+	params *rds.CreateDBSnapshotInput,
+) (
+	output *rds.CreateDBSnapshotOutput,
+	err error,
+) {
+	instanceID := params.DBInstanceIdentifier
+	_, instance, err := m.FindInstance(*instanceID)
```

Comment:
This is a method I added this last week to find out instances in the mock.
The `FindSnapshot` is similar to this, but it's looking for snapshot (based on its id).
These return error on missing the object, or the object.